src/pages/Models/Details/ModelDetails.js

```diff
@@ -376,7 +383,13 @@ const ModelDetails = () => {
           </SlidePanel>
         </div>
       )}
-      {generateTerminalComponent(modelUUID, controllerWSHost)}
+      {showWebCLI ? (
+        <WebCLI
+          controllerWSHost={controllerWSHost}
+          credentials={credentials}
+          modelUUID={modelUUID}
+        />
+      ) : null}
```

Comment:
If you have nothing to show as an alternative, you can simplify this conditional like so;

```
      {showWebCLI &&
        <WebCLI
          controllerWSHost={controllerWSHost}
          credentials={credentials}
          modelUUID={modelUUID}
        />
     }
```

Reply:
done